databases/week1/week1.sql

```diff
@@ -0,0 +1,32 @@
+--1)Find out how many tasks are in the task table
+  SELECT count(id) AS count_task  FROM task;
+
+--2)Find out how many tasks in the task table do not have a valid due date
+SELECT count(id) AS  "count_tasks_with_invalid_due_date" FROM task WHERE due_date IS null;
+
+--3)Find all the tasks that are marked as done
+SELECT * FROM task WHERE status_id = 3;
+
+--4)Find all the tasks that are not marked as done
+SELECT * FROM task WHERE status_id != 3;
+
+--5)Get all the tasks, sorted with the most recently created first
+SELECT * from task ORDER BY created DESC ;
+
+--6)Get the single most recently created task
+SELECT * from task ORDER BY created DESC LIMIT  1 ;
+
+--7)Get the title and due date of all tasks where the title or description contains database
+SELECT title,due_date FROM task WHERE title LIKE '%database%' or DESCRIPTION LIKE '%database%';
+
+
+--8)Get the title and status (as text) of all tasks
+SELECT title, status.name FROM task INNER JOIN status ON status.id = status_id;
+
+--9)Get the name of each status, along with a count of how many tasks have that status
+SELECT status.name,count(task_id) FROM task  INNER JOIN status ON task.status_id= status.id ;
```

Comment:
I'm getting this error when running the query: `Error in query (1054): Unknown column 'task_id' in 'field list'`. Looks like you made a typo in your count.

If if fix that typo i get another error: `Error in query (1140): In aggregated query without GROUP BY, expression #1 of SELECT list contains nonaggregated column 'hyf_lesson1.status.name'; this is incompatible with sql_mode=only_full_group_by`

Can you figure out what is missing in order to get the query to work?